src/file-manager/fm-desktop-icon-view.c

```diff
@@ -387,13 +409,21 @@ realized_callback (GtkWidget *widget, FMDesktopIconView *desktop_icon_view)
 {
     GdkWindow *root_window;
     GdkScreen *screen;
+    GdkDisplay *display;
     GtkAllocation allocation;
-    gint scale;
 
     g_return_if_fail (desktop_icon_view->priv->root_window == NULL);
 
     screen = gtk_widget_get_screen (widget);
-    scale = gtk_widget_get_scale_factor (widget);
+    display = gdk_screen_get_display (screen);
+    if (GDK_IS_X11_DISPLAY (display))
+        root_window = gdk_screen_get_root_window (screen);
+
```

Comment:
remove this empty line

Reply:
Build went OK with that empty line removed, I had been under the impression that it was required after a one line `if()` block not enclosed in brackets

Reply:
It's not, C doesn't care about blanks but for separating tokens. You could write all your C file in one line, that'd be a mess but it'd work just fine -- assuming you could debug it though :D

Reply:
Thanks